app/views/validate.py

```diff
@@ -1,17 +1,22 @@
 import json
+import os
 import urllib
 from json import JSONDecodeError
 
+import requests
 from flask import Blueprint, Response, jsonify, request
 from structlog import get_logger
 
 from app.validators.questionnaire_validator import QuestionnaireValidator
-from app.validators.schema_validator import SchemaValidator
 
 logger = get_logger()
 
 validate_blueprint = Blueprint("validate", __name__)
 
+AJV_HOST = os.getenv("AJV_HOST", "localhost")
+
+AJV_VALIDATOR_URI = f"http://{AJV_HOST}:5002/validate"
```

Comment:
Minor but URL since the protocol is defined.
```suggestion
AJV_VALIDATOR_URL = f"http://{AJV_HOST}:5002/validate"
```